puzzles/views.py

```diff
@@ -724,15 +725,15 @@ def hint_list(request):
             .select_related()
             .order_by('-submitted_datetime')
         )
-        query_description = "Hints"
+        query_description = _("Hints")
         if 'team' in request.GET:
             team = Team.objects.get(id=request.GET['team'])
             hints = hints.filter(team=team)
-            query_description += " from " + team.team_name
+            query_description += _(" from ") + team.team_name
         if 'puzzle' in request.GET:
             puzzle = Puzzle.objects.get(id=request.GET['puzzle'])
             hints = hints.filter(puzzle=puzzle)
-            query_description += " on " + puzzle.name
+            query_description += _(" on ") + puzzle.name
```

Comment:
The ideal way would be to rewrite the code to have 4 translations: "Hints", "Hints from team", "Hints on puzzle", "Hints from team on puzzle", but I'm changing it to be less chopped up by making "from team" and "on puzzle" 2 strings instead of 4. I would think it's flexible enough to be usable in most languages.